docs/ENVIRONMENT.md

```diff
@@ -351,6 +351,11 @@ All tools:
   default is not appropriate for a given system. Defaults to `5`, accepts
   positive integers.
 
+* `$SYSTEMD_DEFAULT_MOUNT_RATE_LIMIT_INTERVAL` — can be set to override the mount
+  units interval rate limit for parsing `/proc/self/mountinfo`. Similar to `$SYSTEMD_DEFAULT_MOUNT_RATE_LIMIT_BURST`, the interval limit may be
+  adjusted when the default is not appropriate for a given system. Defaults to `1`,
+  accepts positive integers.
```

Comment:
should say the time unit applied by default (i.e. seconds), and that the time unit may be overriden as usual by specifying it explicitly, with a reference to the systemd.time(7) man page